examples/complete/README.md

```diff
@@ -1,4 +1,7 @@
-# Complete example
+# Basic example
```

Comment:
Should this be Complete example (as per the directory name)

Reply:
fixed